src/__tests__/notifications.test.ts

```diff
@@ -207,44 +231,44 @@ describe("CourierNotifications", () => {
             id: "block_1d4c32e0-bca8-43f6-b5d5-8c043199bce6",
             type: "text",
             locales: {
-              fr_FR: "block fr 1"
-            }
+              fr_FR: "block fr 1",
+            },
```

Comment:
we have stupid lint rules that don't like trailing commas here so it may complain on the build, don't quite remember if we still have the checks

Reply:
`yarn lint` passed 🤷